sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.0.0-beta.5 (Unreleased)
 
 ### Breaking Changes
+- The whole `DatasourceCredential` API has been renamed to `DataSourceCredential`. This includes renames in types, methods, and properties.
```

Comment:
will this happen across languages too?

Reply:
Nope. .NET is the odd one out this time. Discussed with other languages and Johan, and they're fine with the difference.